src/PowerShellEditorServices/Session/PowerShellContext.cs

```diff
@@ -1700,7 +2148,7 @@ private void OnDebuggerStop(object sender, DebuggerStopEventArgs e)
                     // Write a new output line before continuing
                     this.WriteOutput("", true);
 
-                    e.ResumeAction = localDebuggerStoppedTask.Result;
+                    e.ResumeAction = localDebuggerStoppedTask.GetAwaiter().GetResult();
```

Comment:
Why did you need to make this change?

Reply:
This is closer to how a task is awaited behind the scenes.  Most notably it unwraps the generic `AggregateException` that `Task.Result` throws when the task threw an exception.